app/src/components/menubar/menubar.module.css

```diff
@@ -0,0 +1,50 @@
+#menuBar {
```

Comment:
A lot of really cool styling, from the backdrop filter, to box shadow,  having special styling for first vs last child and hover styling. And the menuBar hover. Shows a lot attention to detail and intiative in learning these css properties!